apps/client/app/components/footer/index.tsx

```diff
@@ -1,10 +1,16 @@
-import * as React from 'react'
+import { Fragment } from 'react'
 import {
   GITHUB_REPO_URL,
   MYLES_PUDO_URL,
   TWITTER_ACCOUNT_URL,
 } from '@/constants/index.ts'
-import {type JSX} from 'react/jsx-runtime'
+import { type JSX } from 'react/jsx-runtime'
+import { Link } from '@remix-run/react'
+
+const insideNavigation = [
+  { name: 'Pricing', href: '/#pricing', routeType: 'link' },
+  { name: 'Terms Of Use', href: '/terms', routeType: 'link' },
+]
```

Comment:
```suggestion
  { name: 'Pricing', href: '/#pricing' },
  { name: 'Terms Of Use', href: '/terms' },
]
```

I don't think you need this anymore, right ?🤔